verifier/verifier_test.go

```diff
@@ -605,3 +605,33 @@ func TestVerifyX509TrustedIdentities(t *testing.T) {
 		})
 	}
 }
+
+func TestVersionCompare(t *testing.T) {
+    var testMinVer1 string = "v0.0.9"
+    var testMinVer2 string = "v1.0.1"
+    var testMinVer3 string = "v1.0.0"
+    var testMinVer4 string = "v1.0.0-alpha"
+    var testMinVer5 string = "v1-pre+meta"
+    var testPlugVer string = "1.0.0"
+
+    err := versionCompare(testMinVer1, testPlugVer)
+    if err != -1{
+        t.Fatalf("TestVersionCompare Error: %d WantErr: -1", err)
```

Comment:
```suggestion
        t.Errrof("TestVersionCompare Error: %d WantErr: -1", err)
```
applicable to all tests

Reply:
Modified